src/pages/faqs/index.js

```diff
@@ -1,15 +1,37 @@
 import React, { Fragment } from 'react';
 import Section from '../../components/styledComponents/Section';
+import FAQ from '../../components/Faq';
+import {FaqStyleContainer} from '../../components/Faq/styles';
 
```

Comment:
Looks like you're missing the graphql import. 

just add this: 
import {graphql} from 'gatsby';